models/product-type/src/product-type-draft/presets/sample-data-fashion/jackets.spec.ts

```diff
@@ -106,9 +106,7 @@ describe(`with jackets preset`, () => {
             ],
             "name": "sleeve_length",
             "type": {
-              "__typename": "enum",
-              "name": "enum",
-              "values": [
+              "enum": [
```

Comment:
It's easier to see what graphql expects when comparing the diff in the snapshot tests 👍